packages/launch/_assets/layout.css

```diff
@@ -119,13 +126,17 @@ hr {
 
 /** HEADER ****************************************************************************************/
 header {
-  border-bottom: 1px solid var(--primary-color-darker);
+  border-bottom: 2px solid var(--primary-lines-color);
   align-items: center;
-  padding: 10px 15px;
-  position: sticky;
+  padding: 25px 0;
   top: 0;
-  background: linear-gradient(to bottom, var(--primary-color-lighter), var(--primary-color));
+  position: sticky;
   z-index: 100;
+  background-color: var(--page-background);
+}
+
+body[layout='home'] header {
+  position: relative;
```

Comment:
For landing pages that barely scroll, I think it's okay to not make it sticky. Which also just looks nicer because when you scroll the background image up, the header becomes really weird looking because it is transparent on the landing page.